src/index.ts

```diff
@@ -89,6 +89,7 @@ export const resolveFetch = () => {
 };
 
 export class UnleashClient extends TinyEmitter {
+    bootstrapped = false;
```

Comment:
Is this essentially a combination of other fields? What does it mean if `bootstrapped` is `true`, but `bootstrap` and `bootstrapOverride` are false? Would it make more sense as a method?

And what's the property used for? I see it shows up in the test, but is it used for anything else?